### DISCUSSION TITLE
Why is creating multiple clusters in the same namespace a bad idea?

### DISCUSSION THREAD

**Discussion by wishkres** [strimzi/strimzi-kafka-operator#8107] [Q&A]
While searching, I found the following discussion: https://github.com/strimzi/strimzi-kafka-operator/discussions/6783

We have already implemented Strimzi, and we currently have two clusters in the same Kubernetes namespace, and both are running the topic operator. We are not using the user operator because we need to maintain users from a different mechanism.

The mentioned discussion says that putting two clusters in the same namespace is a bad idea. I was wondering if I could get more details on why this is a problem? I want to make the recommendation that we switch to separate namespaces, but I was hoping to get more details on what problems the same namespace was causing.

**Comment by scholzj** [comment]
Essentially, each of the clusters will have some topics with the same name. Such as the `__consumer-offsets` etc. In theory, the Topic Operator should rename them when creating the custom resources. But it does not always work on 100% and creates conflicts. These are dangerous since they can result in unwanted topic deletion etc.
